ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -359,84 +364,89 @@ export default class ConfirmTransactionBase extends Component {
 
     if (process.env.SHOW_EIP_1559_UI) {
       return (
```

Comment:
This return block is the same as before the PR, with the exception of the inclusion of the `nonceField` (which conditionally renders if `useNonceField`, which was defined before this PR, is true)